### PR TITLE
upgraded cypress and debugged mapTools.js

### PR DESCRIPTION
**Pull request by vishal5251**
I had upgraded the cypress version and debugged the mapTool.js (path: C:\Users\Vishal\Documents\GitHub\treetracker-wallet-web\src_old\mapTools.js).

### CODE REVIEW

**Comment by dadiorchen** [comment]
@vishal5251 thanks for you work! bty, are you in the slack channel?

**Comment by vishal5251** [comment]
nope..

**Comment by dadiorchen** [comment]
> nope..

We have a community on slack, if you like you can leave you email here or send it to me so we can invite you to join the community!

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -1,28 +1 @@
...
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
-
-  // copy from https://github.com/cypress-io/cypress/tree/master/npm/react
-  require('@cypress/react/plugins/react-scripts')(on, config)
-  // IMPORTANT to return the config object
-  // with the any changed environment variables
-  return config
-}
+module.exports = (on, config) => {};
```
[the remark this answers is not in the document]
Comment (vishal5251) [reply cypress/plugins/index.js:1]: @dadiorchen Yes, I commented the line and it still it works.

**Comment by vishal5251** [comment]
vishaljaiswal9551@gmail.com this is my gmail account.

File: cypress/plugins/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dadiorchen) [reply cypress/plugins/index.js:1]: Good, thank you for you contribution.

**Comment by pvdo** [comment]
image

I am not much familiar with Cypress. @vishal5251 Do you know how to fix this issue?

**Comment by dadiorchen** [comment]
@pvdo did you try to restore these lines:

```
module.exports = (on, config) => {
  // `on` is used to hook into various events Cypress emits
  // `config` is the resolved Cypress config

  // copy from https://github.com/cypress-io/cypress/tree/master/npm/react
  require('@cypress/react/plugins/react-scripts')(on, config)
  // IMPORTANT to return the config object
  // with the any changed environment variables
  return config 
```

**Comment by pvdo** [comment]
That actually works! I should have done that 😑
Thanks @dadiorchen !

**Pull request merged**